docs/resource-categorization.md

```diff
@@ -440,30 +440,33 @@ mf2c-curl-post https://localhost/api/fog-area -d
 
 ### Troubleshooting
 
-- To get the `Host-IP` related information the resource-categorization module is running another container `alpine:latest` . So it might possible that sometimes we may not be able to get the output value from that container.
+- For those who are using `latest-V2.0.16`; To get the `Host-IP` related information the resource-categorization module is running another container `alpine:latest` . So it might possible that sometimes we may not be able to get the output value from that container.
 - Fix the multiple `device_ip` creation issue for a single agent
 
 ## Change LOG
 
+###latest-V2.0.20 (date 19/September/2019)
+
 ###latest-V2.0.18 (date 09/July/2019) [in complience with CIMI-server - 2.20 +later & DataClay version - 2.22 +later]
+
 ### latest-V2.0.17 (date 28/June/2019) [in complience with CIMI-server - 2.20 +later & DataClay version - 2.22]
+
 ### latest-V2.0.16 (date 28/June/2019) [in complience with CIMI-server - 2.18 / 2.19 & DataClay version - 2.21]
 
 
 #### Added
```

Comment:
each new addition and changes must go into the respective new version. See how the other have written theirs. Otherwise we cannot track which release has which changes

Reply:
added the text

Reply:
I don't see the changes...your Added and Changed sections are still in the bottom, in the wrong place. They must go into their respective release version

Reply:
@cjdcordeiro is it okay now?